lib/datasets/diamonds.rb

```diff
@@ -0,0 +1,37 @@
+module Datasets
+  class Diamonds < Dataset
+    Record = Struct.new(:carat,
+                        :cut,
+                        :color,
+                        :clarity,
+                        :depth,
+                        :table,
+                        :price,
+                        :x,
+                        :y,
+                        :z)
+
+    def initialize()
+      super()
+      @metadata.id = "diamonds"
+      @metadata.name = "Diamonds"
+      @metadata.licenses = ["CC0"]
```

Comment:
Where did you find the license information?

Reply:
License information obtained from https://openml.org/d/42225 but no information given at https://ggplot2.tidyverse.org/reference/diamonds.html or in the [GitHub repository](https://github.com/tidyverse/ggplot2/blob/main/R/data.R) if the datasets have a different license than the code

Reply:
Raised issue https://github.com/tidyverse/ggplot2/issues/4681

Reply:
Licensing information is unclear https://github.com/tidyverse/ggplot2/pull/4686#issuecomment-986769199 Is "unknown" ok? The other ggplot2 datasets have a similar format and origin information so can also be used either instead or in addition.

Reply:
Umm...
I don't think that "Unknown" is a good idea. We should not implement download code for "Unknown" license data.

How about "MIT" that is the same license as ggplot2? We can also add this note as a comment.

Reply:
Will try making another pull request to:
a) Update the Presidents terms dataset
b) Add public domain license information to the Fuel Economy dataset, should be public domain according to https://resources.data.gov/open-licenses/ , but checking

Reply:
The library has the following citation information https://github.com/tidyverse/ggplot2/blob/main/inst/CITATION maybe this is sufficient? MIT license only covers code.

Reply:
https://github.com/tidyverse/ggplot2/blob/main/inst/CITATION#L1

> citHeader("To cite ggplot2 in publications, please use:")

It seems that this is for publication.

BTW, it may be better that we add `citation` to our metadata: https://github.com/red-data-tools/red-datasets/blob/master/lib/datasets/metadata.rb